src/app/components/menu/image.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+const Images = ({ imageSrc = null, iconList = [] }) => {
```

Comment:
Usuwasz propsy wszystkie

Reply:
??

Reply:
Tutaj to powinno być tak jak ustaliliśmy

const Images = ({ imageSrc, type }) => {

Reply:
gdzie type = 'image' lub type = 'icon'

a potem if (type === 'icon')